bin/stashcp

```diff
@@ -97,6 +97,14 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
     payload['start2']=start2
     payload['end2']=end2
     
+    with open(".job.ad") as job_file:
+        for line in job_file.readLines():
+            if "AutoClusterAttrs" in line:
```

Comment:
- You are going to have problems with any string that happens to contain these.  Do proper parsing of the line.
- ClassAd attribute names are not be case sensitive, but your parsing code is.